packages/hooks/src/useFocusWithin/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+import useEventListener from '../useEventListener';
+import type { BasicTarget } from '../utils/domTarget';
+
+export interface Options {
+  onFocus?: (e: FocusEvent) => void;
+  onBlur?: (e: FocusEvent) => void;
+  onChange?: (focused: boolean) => void;
```

Comment:
这个参数感觉叫 `hasFocus` 更好一些……？

```suggestion
  onChange?: (hasFocus: boolean) => void;
```

Reply:
或者和返回值保持一致都叫 `isFocusWithin`……？

Reply:
fixed